tools/js-sdk-release-tools/src/autoGenerateInPipeline.ts

```diff
@@ -14,7 +14,7 @@ async function automationGenerateInPipeline(inputJsonPath: string, outputJsonPat
     const inputJson = JSON.parse(fs.readFileSync(inputJsonPath, {encoding: 'utf-8'}));
     const specFolder: string = inputJson['specFolder'];
     const readmeFiles: string[] | string | undefined = inputJson['relatedReadmeMdFiles']? inputJson['relatedReadmeMdFiles']: inputJson['relatedReadmeMdFile'];
-    const cadlProjectFolder: string[] | string | undefined = inputJson['relatedCadlProjectFolder'];
+    const cadlProjectFolder: string[] | string | undefined = inputJson['relatedTypespecProjectFolder'];
```

Comment:
Please also update 'cadlProjectFolder'

Reply:
Also the TypeSpec capitalization should be `relatedTypeSpecProjectFolder`.

Reply:
@MaryGao , please use capitalization. I made the change in sdk automation contract schema.